spring-cloud-gcp-logging/README.adoc

```diff
@@ -26,6 +26,11 @@ Platform.
 This module provides support for associating a web request trace ID with the corresponding log entries.
 This allows grouping of log messages by request.
 
+NOTE: Due to the way logging is set up, the GCP project ID and credentials defined in
+`application.properties` are ignored.
+Instead, you should set the `GOOGLE_CLOUD_PROJECT` and `GOOGLE_APPLICATION_CREDENTIALS` environment
```

Comment:
Doesn't hurt to set both of course, but the credentials file also specifies the project, so using only GOOGLE_APPLICATION_CREDENTIALS works too.

Reply:
Good point, from what I observed, only application-default credentials contain the project ID. Service account credentials don't, which is why I added the `GOOGLE_CLOUD_PROJECT` safeguard here.